docs/tutorials/using_apis/connectivity/mesh_tutorial.md

```diff
@@ -0,0 +1,136 @@
+<h1 id="mesh-tutorial">Mesh tutorial</h1>
+
+## Introduction
+
+Mbed OS supports variety of 802.15.4 based Mesh protocols. There is support for [Wi-SUN](../reference/wisun-tech.html), [Thread](../reference/thread-tech.html) and [6LoWPAN-ND](../reference/6LoWPAN-ND-tech.html) protocols and they all have their own unique characteristics. For example, Thread is designed for Home automation use while Wi-SUN and 6LoWPAN-ND are targeted more for Utilities and Smart metering. In addition, Wi-SUN and Thread are having certification programs to ensure interoperability between devices from different vendors.
```

Comment:
I love this section and how it talks about what the different protocols are designed for. Could you please expand on when to use which protocol?

Reply:
Will be added soon :-)

Reply:
Updated in commit "Update mesh introduction"